_src/pattern-library/sass/components/_buttons.scss

```diff
@@ -57,6 +57,10 @@
 .btn-base {
     padding: spacing-vertical(x-small) spacing-horizontal(base);
     font-size: font-size(base);
+
+    svg {
```

Comment:
Also, are we worried about the case of Text + Icon combination styling here? What if I want an icon to be smaller than my base text? Is that on me as a knowledgable FED person to customize that with a font-size in my individual case?